### PR TITLE
Rai dashboard addition

### PR DESCRIPTION
**Pull request by rkamat25**
# Description

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

File: sdk/python/responsible-ai/text/responsibleaidashboard-text-classification-financial-news/responsibleaidashboard-text-classification-financial-news.ipynb [hunk 1]
```diff
@@ -0,0 +1,2478 @@
...
+     "outputs_hidden": false,
+     "source_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "outputs": [
```
Comment (imatiach-msft) [inline sdk/python/responsible-ai/text/responsibleaidashboard-text-classification-financial-news/responsibleaidashboard-text-classification-financial-news.ipynb:279]: clear notebook outputs

**Comment by imatiach-msft** [comment]
I think csv files should be moved out of github and instead put on blob storage - I can help with this

**Comment by rkamat25** [comment]
@microsoft-github-policy-service agree [company="Fractal Analytics Inc"]

File: sdk/python/responsible-ai/text/responsibleaidashboard-text-classification-financial-news/responsibleaidashboard-text-classification-financial-news.ipynb (hunk 1 above)
Comment (rkamat25) [reply sdk/python/responsible-ai/text/responsibleaidashboard-text-classification-financial-news/responsibleaidashboard-text-classification-financial-news.ipynb]: > clear notebook outputs

Notebook outputs have been cleared

**Comment by rkamat25** [comment]
> I think csv files should be moved out of github and instead put on blob storage - I can help with this
Moved the csv out of the Repo, shared the csv files to be stored at blob. waiting for the links.

**Pull request closed without merging**